react/features/feedback/components/Feedback.web.js

```diff
@@ -0,0 +1,28 @@
+/* global config */
+import React, { Component } from 'react';
+
+/**
+ * A Web Component which renders feedback button.
+ */
+export class Feedback extends Component {
```

Comment:
Since this component doesn't use any of the lifecycle methods nor state you can make it a stateless functional component instead.